motionplan/constraint.go

```diff
@@ -272,6 +272,13 @@ func createAllCollisionConstraints(
 		return nil, err
 	}
 
+	// fmt.Println("inputs: ", inputs)
```

Comment:
[note] found these logs very useful for debugging. Will remove before merging as there is no debug logger I can use to display these in `contrainsts.go`

Reply:
Ultimately we want the RC card to display obstacles I think, or maybe some part of the frame system where it is a resource you can actually pass a logger to.  Good to think about this for useability without having to access package print statements directly.

Reply:
I dont think we want these we should be able to log these from upstream if necessary

Reply:
true, I think the benefit gained for my debugging process was being able to view the obstacles in the world frame which saves time on conversions. Maybe we can support a way to return the worldState in various reference frame?

I will still be removing these before merge